indexes/utxoindex/src/stores/indexed_utxos.rs

```diff
@@ -96,6 +109,22 @@ impl UtxoEntryFullAccessKey {
         bytes.extend_from_slice(transaction_outpoint_key.as_ref());
         Self(Arc::new(bytes))
     }
+
+    /// Extracts a [`ScriptPublicKey`] and [`TransactionOutpoint`] of the  [`UtxoEntryFullAccessKey`]
+    pub fn extract_script_public_key_and_outpoint(&self) -> (ScriptPublicKey, TransactionOutpoint) {
+        let version = ScriptPublicKeyVersion::from_le_bytes(self.0[..VERSION_TYPE_SIZE].try_into().unwrap());
+        let script_size =
```

Comment:
Also not sure why we require script_size here. script could also just be decoded via: `self.0[VERSION_TYPE_SIZE..TRANSACTION_OUTPOINT_KEY_SIZE]`.